entrypoint.py

```diff
@@ -0,0 +1,24 @@
+#!/bin/python3
+
+import os
+import argparse
+import re
+import subprocess
```

Comment:
Order the includes :wink:

Reply:
done in https://github.com/puneetmatharu/cmake-format-lint-action/pull/9/commits/eb98f0c07e7ec986e762bddfcbda39665ef3398b